### ISSUE TITLE
The code is not running properly

### ISSUE THREAD

**Issue by yesod-a** [zhiyuan-ning/GenerativeFL#3]
After passing through gradient descent, there were 0 client selections, causing the code to interrupt prematurely. In my test, a total of 43 rounds were run, and this situation occurred in the 43rd round, leading to termination. Moreover, most of the selected clients were 1 or 2. Could you please tell me what the problem is? I did not modify the model or various parameters. Only oort, tif and fedavg were supplemented for collecting training data.
